python/tskit/trees.py

```diff
@@ -137,6 +159,24 @@ class Individual(util.Dataclass):
     for this individual, decoded if a schema applies.
     """
 
+    @property
+    def population(self) -> int:
+        populations = {self._tree_sequence.node(n).population for n in self.nodes}
+        if len(populations) > 1:
+            raise ValueError("Individual has nodes with mis-matched populations")
+        if len(populations) == 0:
+            return tskit.NULL
+        return next(iter(populations))
```

Comment:
I think ``populations.pop()`` does the same thing, a bit more readably

Reply:
I've changed it to this.